com.github.diegopvlk.Dosage.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.github.diegopvlk.Dosage",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.diegopvlk.Dosage",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libportal",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Ddocs=false",
+                "-Dvapi=false",
+                "-Dbackends=gtk4",
+                "-Dintrospection=true"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/flatpak/libportal/releases/download/0.6/libportal-0.6.tar.xz",
+                    "sha256" : "88a12c3ba71bc31acff7238c280de697d609cebc50830c3766776ec35abc6566"
+                }
+            ]
+        },
+        {
+            "name" : "dosage",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/diegopvlk/Dosage"
```

Comment:
Needs a tag and/or a commit

Reply:
and you'll need to build blueprint since the build happens offline

Reply:
also the appstream file upstream is insufficient. missing `url`, `releases`, `screenshots` and `content_rating`

Reply:
build failure:
```
• tag-missing           : <release> required
```